src/main/java/racingGame/view/InputView.java

```diff
@@ -9,16 +10,22 @@
     private static String[] cars;
 
     public static String[] cars() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
         return inputCarsValidator();
     }
 
     public static int rounds() {
+        System.out.println("시도할 회수는 몇 회 인가요?");
         return inputRoundValidator();
     }
 
     public static String[] inputCarsValidator() {
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/java-racingcar/pull/645#discussion_r399753183

```suggestion
    private static String[] inputCarsValidator() {
```